### DISCUSSION TITLE
Offical AppDB Profile?

### DISCUSSION THREAD

**Discussion by yodaluca23** [qnblackcat/uYouPlus#672] [Q&A] [answered]
Is [this ](https://appdb.to/search/?type=cydia&dev=1900000485)@qnblackcat 's official AppDB profile?
If so, are the apps on it being updated with the latest releases from GitHub?

**Comment by qnblackcat** [comment, marked as answer]
It *was* my account. But I stopped uploading ipa to that site a long long time ago. I recommend you get the ipa from my Github repo

**Reply by yodaluca23** [reply]
Perhaps you might consider keeping them up-to-date once more, especially since [AppDB is now completely free](https://appdb.to/news/459), so a lot of people, myself included, are probably going to switch to AppDB, and this would make it a lot more seamless. Plus, I can't imagine it would be anything too difficult to maintain...

**Reply by qnblackcat** [reply]
I don’t, and will never advertise any paid signing services here

**Reply by yodaluca23** [reply]
> I don’t, and will never advertise any paid signing services here

You don't have to link it in the README or anything but, just making it easier for AppDB users would be nice...
